tests/rules/null-coallesce.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+function f($filename) {
+    // ruleid: AIK_exec-use
+    "convert " . $filename;
```

Comment:
I think the problematic case was using `"convert $filename"` so this is what we should test here